caracal/schema/selfcal_schema.yml

```diff
@@ -153,16 +153,21 @@ mapping:
         required: false
         example: 'I'
       img_multiscale:
-          type: bool
-          desc: Switch on multiscale cleaning.
-          required: false
-          example: 'False'
+        type: bool
+        desc: Switch on multiscale cleaning.
+        required: false
+        example: 'False'
       img_multiscale_scales:
-          seq:
-            - type: int
-          desc: Scales of multiscale [0,10,20,etc.] in pixels.
-          required: false
-          example: '0, 5, 10, 20'
+        seq:
+          - type: int
+        desc: Scales of multiscale [0,10,20,etc.] in pixels.
+        required: false
+        example: '0, 5, 10, 20'
+      img_paralldeconv:
+        desc: Speed-up deconvolution by deconvolving subimages in parallel. Subimages will be at most of the size given by this parameter. If set to 0 no parallel deconvolution is performed.
+        type: int
+        required: false
+        example: '900'
```

Comment:
Sorry to jump in; this is from the wsclean wiki:  "Values of 1024-4096 work well for large images. Smaller values would split the image in very many subimages, increases the computational cost of splitting" - the default image size for is 1800 - which is not very big? Do we need a smaller value here?

Reply:
Frankly, I have no idea what is a reasonable default. Probably  there is none (other than 0 = no parallel deconvolution).

We could turn it around and ask CARACal users to set the number N of subimages they  want to split their image into for parallel deconvolution. Then we would  internally set parallel-deconvolution = npix/sqrt(N). What  do you think?

Reply:
More rope? :)
Can't we use your ncpu global setting and the image size used for the run to automatically estimate the number of  subimages- the user can then always override it if they want?

Reply:
nice idea